vendor/github.com/fsnotify/fsevents/wrap.go

```diff
@@ -229,6 +229,7 @@ func (es *EventStream) start(paths []string, callbackInfo uintptr) {
 	go func() {
 		runtime.LockOSThread()
 		es.rlref = CFRunLoopRef(C.CFRunLoopGetCurrent())
+		C.CFRetain(C.CFTypeRef(es.rlref))
```

Comment:
Good find. Does the upstream project need this too? If so can you please open a PR to contribute this back.

Reply:
Sure